docs/_templates/launch.html

```diff
@@ -13,7 +13,7 @@
 </a>
 
 <a
-  href="{{ pathto('_static/retro/index.html', 1) }}"
+  href="{{ pathto('_static/retro/index.html', 1) }}?path=notebooks/intro-retro.ipynb"
```

Comment:
The format is slightly different for retro:

```suggestion
  href="{{ pathto('_static/retro/notebooks/index.html', 1) }}?path=intro-retro.ipynb"
```